from opal.core.exceptions import Error


class BatchLoadError(Error):
    pass
